tests/unit_tests/data_steward/utils/participant_summary_requests_test.py

```diff
@@ -159,6 +172,36 @@ def test_get_deactivated_participants(self,
 
         self.assertEqual(expected_response, dataset_response)
 
+    @mock.patch('utils.participant_summary_requests.store_participant_data')
```

Comment:
Not sure you need this at all.

Reply:
This is definitely for the ticket where we store the dataframe, I will remove it for now ☺️